core/domain/search_services.py

```diff
@@ -182,7 +182,8 @@ def search_explorations(query, limit, cursor=None):
               web-safe string that can be used in URLs.
     """
     return search_services.search(
-        query, SEARCH_INDEX_EXPLORATIONS, cursor, limit, ids_only=True)
+        query, SEARCH_INDEX_EXPLORATIONS, offset=cursor,
```

Comment:
Please avoid writing stuff like offset=cursor or size=limit. Use consistent language everywhere, otherwise it creates confusion.

(For example, per a separate previous comment, remember that we think of "offset" as an int and "cursor" as an encoded string.)

Reply:
Done in https://github.com/oppia/oppia/pull/11295/commits/93a123cc7869c29b5a8b9e86d62b7fecb13f7a5a.